queries/html_tags/injections.scm

```diff
@@ -64,3 +64,11 @@
       (attribute_value) @regex
     ] (#eq? @_attr "pattern")))
 ))
+
+; <script type="${someMimetypeOrConfiguredType}">
+(script_element
+   (start_tag
+      ((attribute
+           (attribute_name) @_attr (#eq? @_attr "type")
+           (quoted_attribute_value (attribute_value) @_type))))
+   (raw_text) @language (#set-lang-from-mimetype! @_type))
```

Comment:
Can you also remove the <script type ...> from some lines above this? This seems to override that already anyways

Reply:
And maybe add some tests :)
Just to be sure

Reply:
- removed superfluous query
- explicitly associated "module" script type with javascript grammar
- added test cases to html injections test fixture
- added an ecma test fixture
- added some more descriptive comments to html_tags injections query file
- rebased

Reply:
Great! I'll merge later today, just hoping Hamsta will maybe see this and go over it...